share/style.css

```diff
@@ -1,5 +1,6 @@
 /* Resets for extra padding in the window */
-html, body {
+html,
+body {
```

Comment:
ill add linter or something another time for now just ignore this (its mostly copypaste anyway)